index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <link
+    href="https://fonts.googleapis.com/css?family=Open+Sans:400,600,300"
+    rel="stylesheet"
+    type="text/css"
+  />
+  <link rel="stylesheet" href="style.css" />
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Job Board App</title>
+  </head>
+  <body>
+    <ul>
+      <li><a href="default.asp">Home</a></li>
+      <li><a href="news.asp">Blah</a></li>
+      <li><a href="contact.asp">Blah</a></li>
+      <li><a href="about.asp">Blah</a></li>
+    </ul>
+    <p class="title">Job Board App</p>
+    <p>Welcome To The Board</p>
+    <p class="chart">This is where the chart will go
+        <table>
+            <tr>
+                <th>INPUT</th>
+                <th>OUTPUT</th>
+            </tr>
+            <tr>
+                <td>TEXT</td>
+                <td>TEXT</td>
+            </tr>
+        </table>
```

Comment:
tables are not super accessible: They're bad for SEO and screen-readers, in particular

https://www.thoughtco.com/dont-use-tables-for-layout-3468941#:~:text=a%20good%20idea.-,Tables%20Are%20Not%20Accessible,and%20top%2Dto%2Dbottom.

How else can we do this?

Reply:
created new table using ul/li along with css to make more SEO friendly